Examples/test-suite/r/Makefile.in

```diff
@@ -20,7 +20,8 @@ CPP_TEST_CASES += \
 	r_double_delete \
 	r_overload_array \
 	r_sexp \
-        r_overload_comma
+	r_overload_comma \
+	r_memory_leak
```

Comment:
Please keep in alphabetical order and there ought to be a trailing \ on all lines